python/lib/dependabot/python/package_manager.rb

```diff
@@ -0,0 +1,39 @@
+# typed: strong
+# frozen_string_literal: true
+
+require "sorbet-runtime"
+require "dependabot/python/version"
+require "dependabot/ecosystem"
+require "dependabot/python/requirement"
+
+module Dependabot
+  module Python
+    ECOSYSTEM = "Python"
+    PACKAGE_MANAGER = "Python"
```

Comment:
Is package manager `Pyhon`? That's little confusing. I was expecting to have package manager such as `pip` and so on?

Reply:
@kbukum1 , i have added poetry for collection